zio-http/src/main/scala/zhttp/http/Response.scala

```diff
@@ -26,6 +32,19 @@ case class Response[-R, +E] private (
   def addCookie(cookie: Cookie): Response[R, E] =
     self.copy(headers = self.headers ++ List(Header.custom(HttpHeaderNames.SET_COOKIE.toString, cookie.encode)))
 
+    def defaultHealders: Response[R,E] = self.headers.map(_.name.toString.toLowerCase()).filter(_=="content-length") match {
```

Comment:
It should be `defaultHeaders` :)

Reply:
Looks like this is changes required for https://github.com/dream11/zio-http/issues/448